docs/references/general/grid.md

```diff
@@ -0,0 +1,104 @@
+<!-- Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan 
+author(s): Marina Schmidt 
+
+Disclaimer: License under GNU v3.0, the file is found in the root directory under LICENSE 
+
+-->
+# GRID files 
+
+GRID files are post-processed data produced from FITACF files.
+
+## Naming Conventions
+
+### Single radar GRID files
+
+Currently the common naming convention for a single radar GRID file is:
+
+> YYYYMMDD.HH.mm.ss.<3-letter abbreviation>.grid
+
+Some radars provide separate data files for each channel. In this case, the channel is specified after the 3-letter station ID:
+
+> YYYYMMDD.HH.mm.ss.<3-letter abbreviation>.[a-d].grid
+
+See [fitacf](fitacf.md) documentation for more information on this naming convention. 
+
+If a GRID file is produced for 24-hours using the `-c` option in `make_grid` (see [`make_grid` tutorial](../../user_guide/make_grid.md)) then the common naming convention is:
+
+> YYYYYMMDD.<3-letter abbreviation>.grid
+
+### Combined "GRD" files for each hemisphere
+
+After GRID files are produced for separate radars, they may then be combined with other radars' GRID files to make a GRD file. This file has the same format as a GRID file, but includes data from multiple radars. 
+See [combine_grid](../../user_guide/make_grid.md) on how to make GRD files. 
+
+Combined GRD files are produced for the northern and southern hemispheres separately and usually contain 24 hours of data. One naming convention for 24-hr GRD files could be:
+
+> YYYYMMDD.north.grd  
+> YYYYMMDD.south.grd  
+
+## Fields
```

Comment:
we did this with FitACF docs as well can we look at this again when we do RAWACF and then adjust accordingly? 

Would like this to be with 4.4 release

Reply:
Sorry I'm just catching up on these and coming out being bogged down with lots of other things.  Certainly understand that bolding the `Units` and `Data Type` was done before, but I guess I'm curious as to what was the thinking on bolding these?  Bolding the None's in the `Units` does highlight that it doesn't have any units, but having all of the `Data Type` entries bolded doesn't seem to help emphasize things.  I don't think this is major, and if it was just a 'thought this would look best' then that's OK.  We can certainly revisit for a future version.

Reply:
I was trying to do `latex` font at first, and it failed. I wanted to differ it a bit to make it pop. I get asked a lot what the units are, and personally, as an outsider, I don't know either, and it's usually the thing you want to see if you are converting to something. I cannot say how much time-frequency has gotten me. 

`Data Type` not sure why ... might have just something I thought looked nice, or if someone was programming might want to know the accuracy with regards to the data. Especially since `CHAR` is an `int8`, and that threw me through a loop. 

I agree field names and descriptions are essential, but once you know what you want, you have to work/program/convert the data, and that is the thing people will be going back to and trying to figure out. So from my perspective, that seemed a lot more significant as that is what I do a lot. Again personal preference no real care, but for consistency sake, we should hold this off now and review a bit later, especially how this might get moved somewhere else.

Reply:
@mts299, thanks for taking the time to explain things here.  I can understand what you're talking about on the perspective and what my be more desired to look at quickly.  I think you're right on keeping things consistent and we can address this at another time.